OlinkAnalyze/tests/testthat/test-pca_plot.R

```diff
@@ -218,6 +224,18 @@ test_that("minimal PCA plot", {
 
   vdiffr::expect_doppelganger("PCA plot - label outliers", pca_plot_outliers[[1]])
 
+  #Removing Index dependence in PCA plot
+  ids <- sample(npx_data1$SampleID, 100, replace = T)
+  ids <- gsub("A|B","",ids)
+  ids <- unique(sort(as.numeric(ids)))
```

Comment:
There is a control sample in the list of ids at this stage which is not numeric but is called "CONTROL_SMPLE_S" which leads to a warning here...